ext/reflection/tests/ReflectionClassConstant_basic1.phpt

```diff
@@ -53,7 +53,7 @@ reflectClassConstant($instance, "BAD_CONST");
 Reflecting on class constant TestClass::PUB
 
 __toString():
-string(35) "Constant [ public bool PUB ] { 1 }
+string(35) "Constant [ public true PUB ] { 1 }
```

Comment:
This seems to be the only suspicious change for me due to the current lack of `true` type, so I'm ok with changing it back in case of disagreement.

Reply:
Now that we have the ``true`` type this is fine.